cmd/socat/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	cmd := newRootCmd()
+	err := cmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func newRootCmd() *cobra.Command {
+	var uniDir bool
+	cmd := cobra.Command{
+		Use:   "socat <address> <address>",
```

Comment:
Might be worth calling this tcpcat to avoid confusion with the actual socat

Reply:
But this deliberately mimics `socat` interface.